Library/ReportProjectInfoListItem.swift

```diff
@@ -84,7 +84,7 @@ let reportSpamSubListItems = [
   ),
   ReportProjectInfoListItem(
     type: .child,
-    flaggingKind: .abuse,
+    flaggingKind: .guidelinesAbuse,
```

Comment:
@scottkicks This is the one material change here - `NonDeprecatedFlaggingKind` does not include `.abuse` but it does include `.guidelinesAbuse`. What are these enums ultimately used for? Does it matter?

Reply:
I just tested it on staging and was able to submit a report - I tried IP and Abuse. Both look sane:
<img width="1034" alt="Screenshot 2024-03-19 at 4 48 28 PM" src="https://github.com/kickstarter/ios-oss/assets/146007185/5bbb8daa-7b3a-45b4-b7e6-9d5b23b0c64b">

Reply:
They're used to determine the category reason why users are reporting. I think this should be fine